src/molecules/TodoList/TodoList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import TodoItem from '../TodoItem/TodoItem';
+
+import "./TodoList.css"
+
+const TodoList = ({todoList, onTodoDelete}) => {
+  
+    return (
+      <>
+        {
+            todoList.map((todoVal, index)=>{
+                return (
+                    <TodoItem
+                        key={index}
+                        todoText={todoVal}
+                        onTodoDelete={() => onTodoDelete(index)}
```

Comment:
You should avoid inline functions. Always create a named function and use it, IT helps in debugging and also easy t find in call stack.